packages/atlas/src/components/Avatar/Avatar.styles.ts

```diff
@@ -22,14 +22,50 @@ type ContainerProps = {
   size: AvatarSize
   isLoading?: boolean
   isClickable: boolean
-  withoutOutline?: boolean
   // allow passing 'type' prop to Container because it can be rendered as 'button' depending on context
   type?: 'button'
 }
 
-type EditOverlayProps = {
-  size: Omit<AvatarSize, 'default'>
-}
+export const IconAndOverlayWrapper = styled.div`
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  border-radius: 100%;
+  z-index: 3;
```

Comment:
Can we use `z-index` from const? I noticed that you are hardcoding z-indexes in several places in this file.

Reply:
I changed that to `1`, and removed the z-indexes where it was redundant. I would also not opt to use `${zIndex.someproperty}`, because this particular z-Index matters only locally in the context of the Avatar component itself.